src/main/java/Duke.java

```diff
@@ -1,10 +1,130 @@
+import java.util.Scanner;
+import java.util.Arrays;
+import java.lang.String;
+
 public class Duke {
+    public static Task[] taskList = new Task[100];
+    public static int taskCount = 0;
+    public static final String LINE_HEADER = "\n=====================================================\n";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String inputCommand;
+        printGreeting();
+
+        // Command handler
+        do {
+            Scanner in = new Scanner(System.in);
+            inputCommand = in.nextLine();
+
+            String taskObj = new Scanner(inputCommand).next();
+
+            switch (taskObj){
+                case "list":
+                    listAllTasks();
+                    break;
+
+                case "bye":
+                    printGoodbye();
+                    break;
+
+                case "done":
+                    String[] commandWords = inputCommand.split(" ");
+                    int taskNumber = Integer.parseInt(commandWords[1])-1;
+                    markTaskAsDone(taskNumber);
+                    break;
+
+                case "todo":
+                    String taskDetails = inputCommand.substring(4);
+                    addTask(new ToDo(taskDetails));
+                    break;
+
+                case "deadline":
+                    taskDetails = inputCommand.substring(8);
+                    String[] detailsWords = taskDetails.split("/by ");
+                    addTask(new Deadline(detailsWords[0], detailsWords[1]));
+                    break;
+
+                case "event":
+                    taskDetails = inputCommand.substring(5);
+                    detailsWords = taskDetails.split("/at ");
+                    addTask(new Event(detailsWords[0], detailsWords[1]));
+                    break;
```

Comment:
For the "switch" statement, it would be better if you provide a "default" case also.